src/Lecture1.hs

```diff
@@ -98,7 +113,11 @@ start position can be considered as zero (e.g. substring from the
 first character) and negative end position should result in an empty
 string.
 -}
-subString start end str = error "TODO"
+subString :: Int -> Int -> [Char] -> [Char]
+subString start end str
+        | end < 0   = ""
+        | start <= 0 = take (end+1) (drop (start-start) str)
```

Comment:
`start - start` equals to zero and `drop 0` doesn't change the list. So you can just apply `take` to the string in this case 
```suggestion
        | start <= 0 = take (end + 1) str
```

Reply:
🤦 Fixed.